infra/cifuzz/actions/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Docker image to run CIFuzz in.
+FROM debian:9.5-slim
```

Comment:
why not `ubuntu:16.04` as all other our images?

Reply:
Done.